### DISCUSSION TITLE
How do I configure and select Spyder as an editor into Github Desktop?

### DISCUSSION THREAD

**Discussion by richlysakowski** [community/community#37919] [Other Feature Feedback, Questions, & Ideas]
I need to know how to configure and select editors into Github Desktop that are not currently supported by Microsoft.

I want to configure and select Spyder as an editor into Github Desktop?

Where can I find the internals document that discusses configuration of the pre-integrated editors (RStudio, PyCharm, Notepad++, etc.) ??

**Comment by antonblr** [comment]
Hi @richlysakowski - here is a couple of links that should help:

* source code for win32 supported editors - https://github.com/desktop/desktop/blob/development/app/src/lib/editors/win32.ts
* Readme on how to add new editor support - https://github.com/desktop/desktop/blob/development/docs/technical/editor-integration.md#open-external-editor-integration
* documentation and the list of [Supported Editors - Windows](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/configuring-and-customizing-github-desktop/configuring-a-default-editor?platform=windows#supported-editors)

**Comment by steveward** [comment]
@richlysakowski Spyder support should be landing in https://github.com/desktop/desktop/pull/15542. You can follow along there for updates. I hope that helps!
